### PR TITLE
Removing redundant code, adding jwks_path property

### PR DESCRIPTION
**Pull request by sberyozkin**
I've removed some code which was marked for the deletion, and also remove the doc references to the secret properties since it is not tested right now, so I don't know if we can doc it right now. I propose to doc only those properties which are being tested. 

Thanks

### CODE REVIEW

**Comment by sberyozkin** [comment]
Note credentials.secret property is still obviously supported but until we have a test confirming it is supported at the Vertx level with a test, it is probably better not to type it everywhere :-) Pedro may help with a test so we'll make it visible asap

File: extensions/oidc/runtime/src/main/java/io/quarkus/oidc/VertxKeycloakRecorder.java [hunk 1]
```diff
@@ -25,7 +22,14 @@ public void setup(OidcConfig config, RuntimeValue<Vertx> vertx, BeanContainer be
...
         options.setSite(config.authServerUrl);
         // RFC7662 introspection service address
-        options.setIntrospectionPath(config.introspectionPath);
+        if (config.introspectionPath.isPresent()) {
+            options.setIntrospectionPath(config.introspectionPath.get());
+        }
+
+        // RFC7662 JWKS service address
+        if (config.jwksPath.isPresent()) {
+            options.setIntrospectionPath(config.jwksPath.get());
```
Comment (stuartwdouglas) [inline extensions/oidc/runtime/src/main/java/io/quarkus/oidc/VertxKeycloakRecorder.java]: This looks like a copy/paste error?

Comment (sberyozkin) [reply extensions/oidc/runtime/src/main/java/io/quarkus/oidc/VertxKeycloakRecorder.java]: It does, thanks for catching it

Comment (sberyozkin) [reply extensions/oidc/runtime/src/main/java/io/quarkus/oidc/VertxKeycloakRecorder.java]: Funny, I was staring at the code yesterday evening for a long time :-)

**Comment by sberyozkin** [comment]
@stuartwdouglas, @gsmet Not sure if it is already too late for 0.24.0, that temp code was really my target, I'll add a 0.24.0 milestone, Guillaume, please move it to 0.25.0 if it is too tight

**Review by gsmet — APPROVED** [review]
Looks OK to me now, let's merge.

**Pull request merged**